api/magic_links/routes.py

```diff
@@ -89,13 +89,14 @@ def use(self, link_id: str):
             # else if no link exists (or it has been used)
             # but the user is already logged in
             # then redirect them to the global redirect url
+            frontend_account_url = f"{Config.MAGIC_LINK_REDIRECT_URL}?fund={fund_short_name}&round={round_short_name}"
```

Comment:
Do we need to check if these are present or does the endpoint we redirect to do that?

Reply:
If these are not present then it'll just render all funds (like we do atm) i.e http://localhost:3008/account?fund=&round=

Although because this link is created via magic link page I don't think they will ever be missing, unless the user does it themselves!